packages/astro/src/assets/utils/svg.ts

```diff
@@ -22,7 +22,7 @@ export type SvgRenderMode = 'inline' | 'sprite';
 export function makeSvgComponent(
 	meta: ImageMetadata,
 	contents: Buffer | string,
-	options?: { mode?: SvgRenderMode },
+	options?: { mode: SvgRenderMode },
```

Comment:
Having an optional `mode` should be a bug. Users should either provide a boolean, or provide an object with explicit `mode`